GVFS/GVFS.FunctionalTests/Tests/FastFetchTests.cs

```diff
@@ -26,7 +26,7 @@ public class FastFetchTests
         public void InitControlRepo()
         {
             Directory.CreateDirectory(this.fastFetchControlRoot);
-            GitProcess.Invoke("C:\\", "clone -b " + Settings.Default.Commitish + " " + GVFSTestConfig.RepoToClone + " " + this.fastFetchControlRoot);
+            GitProcess.Invoke(this.fastFetchControlRoot, "clone -b " + Settings.Default.Commitish + " " + GVFSTestConfig.RepoToClone + " " + this.fastFetchControlRoot);

```

Comment:
The intended behavior of the original code was to run the command outside of any repo, which is why it was using the `C:\` drive root. I don't think we should pass in the control root. Any path outside of it (the parent directory?) would do fine.